atlasaction/action.go

```diff
@@ -91,6 +93,29 @@ func MigratePush(ctx context.Context, client *atlasexec.Client, act *githubactio
 	return nil
 }
 
+func MigrateLint(ctx context.Context, client *atlasexec.Client, act *githubactions.Action) error {
+	ghContext, err := createContext(act)
+	if err != nil {
+		return fmt.Errorf("failed to read github metadata: %w", err)
+	}
+	buf, err := json.Marshal(ghContext)
+	var resp bytes.Buffer
+	err = client.MigrateLintError(ctx, &atlasexec.MigrateLintParams{
+		DevURL:    act.GetInput("dev-url"),
+		DirURL:    act.GetInput("dir"),
+		ConfigURL: act.GetInput("config"),
+		Env:       act.GetInput("env"),
+		Context:   string(buf),
+		Web:       true,
+		Writer:    &resp,
+	})
+	if err != nil {
+		return err
+	}
```

Comment:
if the error is that there are lint errors, we need to still set the output 
also add a test for that